Platform/QemuBoardPkg/CfgData/CfgDataDef.dsc

```diff
@@ -141,10 +141,25 @@
   # !BSF HELP:{Silicon Test 1}
   gCfgData.SiliconTest1           |      * | 0x04 | 0x11223347
 
-  # !BSF NAME:{Silicon Test 9}
-  # !BSF TYPE:{EditNum, HEX, (0x00000000,0xFFFFFFFF)}
-  # !BSF HELP:{Silicon Test 9}
-  gCfgData.SiliconTest2           |      * | 0x04 | 0x11223348
+  # !BSF NAME:{Silicon Test 2}
+  # !BSF TYPE:{Table}
+  # !BSF OPTION:{ 0:1:HEX, 1:1:HEX, 2:1:HEX, 3:1:HEX}
+  # !BSF HELP:{Silicon Test 2 to show BYTE table configuration}
+  gCfgData.SiliconTest2           |      * | 0x04 | 0x04030201
```

Comment:
0x04030201 is test purpose? It should be { 0x01, 0x02, 0x03, 0x04 }. Support both way?

Reply:
Yes, it is for test.  It supports both array format {0x01, 0x02, 0x03, 0x04}, and  DWORD format 0x04030201